src/koaApi.js

```diff
@@ -40,12 +40,17 @@ export function setupApp(done) {
 
   // Configure Sessions Middleware
   app.keys = [config.api.sessionKey]
+  
+  if (config.api.trustProxy) {
+    app.proxy = true;
```

Comment:
```suggestion
    app.proxy = true
```
I would just say to remove the terminating ; as the convention is to not user line terminators in openhim-core